mailserver/mailserver.go

```diff
@@ -638,3 +638,7 @@ func (s *WMailServer) bloomFromReceivedMessage(msg *whisper.ReceivedMessage) ([]
 
 	return msg.Payload[8 : 8+whisper.BloomFilterSize], nil
 }
+
+func peerIDString(peer *whisper.Peer) string {
+	return string(peer.ID())
```

Comment:
how that string would be encoded? i think it is better to print peer.peer.ID - which would the same as in enode url

Reply:
https://github.com/ethereum/go-ethereum/blob/v1.8.16/p2p/discover/node.go#L234

Reply:
That is better! 👍